README.md

```diff
@@ -1,11 +1,32 @@
 # Containerized Kubelet
 
-This repository contains Docker images for kubelet 1.18+. Both amd64 and arm64 are provided as
-multiarch images.
+This repository contains Docker images for all kubelet versions currently supported by upstream.
+Both amd64 and arm64 are provided as multiarch images.
```

Comment:
We're not removing old images and there might be users who still use older images, so we should reword this a little bit. I'd say that we maintain images for versions currently supported by the upstream, but images for older versions might be available as well. The table below could show images that are actively maintained, with a note to check Quay for the list of available images. This is similar to what we already have, just slightly reworded.